### DISCUSSION TITLE
Domain change not working

### DISCUSSION THREAD

**Discussion by chiva** [nextcloud/all-in-one#3860] [Questions] [answered]
Hello!

I wanted to change the domain of an existing Nextcloud instance I'm running, but it is not working.

I have followed the indications on the [README](https://github.com/nextcloud/all-in-one#how-to-change-the-domain):
* Edited the `configuration.json` file and wrote my new domain on the `domain` key.
* Stopped containers
* Started containers

After that, if I use my new domain, I get sent to the old one, which is working fine, but the Nextcloud instance is still using the old domain URLs instead of switching to the new one.
It is as if the configuration file was changed, but the changes didn't make it into the containers.

Did I miss any steps?
Would containers need to be recreated to get the new config values? How could I do that?

Thanks!

**Comment by szaimen** [comment, marked as answer]
Hi, it looks like you did not restart the containers via the AIO interface after changing the domain?

**Reply by chiva** [reply]
That was it!
Didn't know that stopping/starting the containers through the AIO interface did trigger a rebuild, and was different than manually starting/stopping the containers.

Thanks a lot!
